tests/pthread/call_sync_on_main_thread.js

```diff
@@ -6,7 +6,7 @@ mergeInto(LibraryManager.library, {
   getDomElementInnerHTML__sig: 'viii',
   getDomElementInnerHTML: function(domElementId, dst, size) {
```

Comment:
Looks good, thanks! But please rename this method so the name isn't misleading, to say `getParentInnerHTML`.

Reply:
Done.